scripts/build-testing-sdk.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
Any scripts that are directly executed, I would remove the `.js` and add a shebang.